config/src/main/java/org/axonframework/config/AggregateConfigurer.java

```diff
@@ -304,6 +308,19 @@ private String name(String prefix) {
         this.filterEventsByType.update(filterConfigurationPredicate);
         return this;
     }
+	
+	/**
+     * Configures caching for the Aggregate type under configuration.
+     * <p>
+     * Note that this configuration is ignored if a custom repository instance is configured.
+     *
+     * @param cache The function creating the Cache
+     * @return this configurer instance for chaining
+     */
+    public AggregateConfigurer<A> configureCache(Function<Configuration, Cache> cache) {
```

Comment:
We do not have a direct `AggregateConfigurerTest`, but we do have the `DefaultConfigurerTest` which uses th `AggregateConfigurer` for testing purposes. 

I'd thus like to ask you to add a test there, which verifies you've configured an Aggregate to use a Cache.
Then, you can request the configuration for the `Configuration#repository(Class<T>)` method to retrieve a `Repository` for said Aggregate, which thus should be of type `CachingEventSourcingRepository`.